universal-login-react/src/core/utils/isPrefilledAmountUsedBy.ts

```diff
@@ -0,0 +1,7 @@
+import {TopUpProvider} from '../models/TopUpProvider';
+
+const providersUsedPrefilledAmount = [TopUpProvider.RAMP];
```

Comment:
`providersRequiresAmount`

Reply:
I understood that the provider does not require amount but it is possible to pass the amount. `providersUsedPrefilledAmount` is not the best name also.